src/main/resources/META-INF/gradle-plugins/com.google.cloud.tools.appengine-appenginewebxml.properties

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
What's this file for?

Reply:
it just allows the user to use "appengine-appenginewebxml" as a plugin
```
apply plugin: 'appengine-appenginewebxml'
```